explainer.md

```diff
@@ -118,6 +136,22 @@ dictionary ItemDetails {
   DOMString description;
   PaymentCurrencyAmount price;
 };
+
+dictionary PurchaseDetails {
+  DOMString itemId;
+  DOMString purchaseToken;
+  boolean acknowledged;
+  PurchaseState purchaseState;
+  // Timestamp in ms since Unix epoch.
+  DOMTimeStamp purchaseTime;
+  boolean willRenew;
```

Comment:
I'd like to rename this to "`autoRenew`" ? WDYT?

Hard to explain but I didn't really understand what "willRenew" would do without referring to your email thread. "auto-renewing subscription" is a pretty self-explanatory concept.

Reply:
I think it's clearest to have a "is" or "will" prefix for booleans. I think willRenew is clear to me (if a sub is cancelled it will not renew).  To me the "auto" is rendundant because there is no other option. All renewals are automatic.

So my brain would stack rank:
 1 - willRenew
 2 - willAutoRenew
 3 - isRenewing
 4 - isAutoRenewing (google play billing's current term)

For history, I believe Play Billing used to have both Manual and Auto renewals, something which is no longer the case. Hence the "Auto", which is now obsolete. However, if it is clearer for some with the Auto in, it's fine with me to leave it (willAutoRenew).

Reply:
willAutoRenew sounds good to me, and it a sort of compromise between the views here. Going with that.